### ISSUE TITLE
SyntaxError: Unexpected end of input

### ISSUE THREAD

**Issue by AdamBrodzinski** [kadirahq/meteor-graphql#9]
I'm getting a weird error now and i'm not sure what's going on.... perhaps my packages are corrupt? All of a sudden none of my Meteor GraphQL apps are working. This is in a new test app after accessing the system info query:

```
W20160110-23:56:59.405(-5)? (STDERR) SyntaxError: Unexpected end of input
W20160110-23:56:59.406(-5)? (STDERR)     at Object.parse (native)
W20160110-23:56:59.406(-5)? (STDERR)     at _GraphQL$types (server/ide.js:27:34)
W20160110-23:56:59.406(-5)? (STDERR)     at doCall (packages/meteorhacks_picker/packages/meteorhacks_picker.js:106:1)
```

Update, I also removed my `~/.meteor` folder and re-installed meteor. Booted up a fresh empty app and am getting the same error.

```
{
  sysInfo {
    hostname
    memory
    uptime
  }
}
```

`http://localhost:3000/graphql/ide?schema=SysInfo`

<img width="1460" alt="screen shot 2016-01-11 at 12 17 39 am" src="https://cloud.githubusercontent.com/assets/1445367/12227341/19c5a614-b7f9-11e5-8f00-beadd27cf082.png">

<img width="669" alt="screen shot 2016-01-11 at 12 20 37 am" src="https://cloud.githubusercontent.com/assets/1445367/12227344/2da67596-b7f9-11e5-8f19-026a8966d3cf.png">

**Comment by AdamBrodzinski** [comment]
Looking deeper into the source it looks like the `variables = '{}'` bit won't work if the variables are an empty string, eg. if you add a variable then take it away.

 I've made a PR that fixes this by letting it be undefined and then falling back to an empty object if so (instead of parsing).

**Comment by arunoda** [comment]
Fixed by #10
